test/Microsoft.AspNet.Mvc.IntegrationTests/FormCollectionModelBindingIntegrationTest.cs

```diff
@@ -147,7 +147,6 @@ private class Address
             // No data is passed.
             var operationContext = ModelBindingTestHelper.GetOperationBindingContext(request =>
             {
-                request.ContentType = "multipart/form-data";
```

Comment:
Because HttpAbstractions threw when there was no data passed. Wasnt interesting for the test case to succeed.